### ISSUE TITLE
Add tokens_keep and dfm_keep?

### ISSUE THREAD

**Issue by koheiw** [kbenoit/quanteda#1037]
We have `tokens_remove()`, which is a shortcut to `tokens_select(x, selection = 'remove')`, but not `tokens_keep()`. It makes our life easier, especially with regex patters.

For example, we have to write to keep capitalized words
```r
tokens_select(toks, '^[A-Z]', 'keep', 'regex')
# or 
tokens_select(toks, '^[A-Z]', valuetype = 'regex')
```
These become just
```r
tokens_keep(toks, '[^A-Z]', 'regex')
```
The same apply to `dfm_select()`.

Am I too lazy?

**Comment by kbenoit** [comment]
You're right, it's consistent, but also a slippery slope: we would then need `tokens_keep()`, and `fcm_keep()`. But hey, why not. Look at how many functions **stringi** has!

**Comment by koheiw** [comment]
Short cut functions do not harm anyone as far as they are consistent.
